test/index.html

```diff
@@ -39,5 +39,13 @@ <h2>Navigation: Only a single navigation element should exist</h2>
     <section>
       <h2>HTML: html tag should have a lang attribute</h2>
     </section>
+    <section>
```

Comment:
This is my bad for not creating PR templates yet, but could you take a screenshot of this and add it to your PR description? 🙂

Reply:
I'm new to contributing, does screenshot mean the picture of this code segment?

Reply:
No worries 😊 If you open `test.html`, could you take a picture of what you see in this specific section of the webpage? We'd like to see what the work you've done looks like on a web page. https://www.google.com/search?q=how+to+take+a+screenshot